healthtools/search/nurses.py

```diff
@@ -21,7 +21,7 @@ def get_nurses_from_nc_registry(query):
     nurses = {'hits': [], 'total': 0}
     try:
         response = requests.get(url)
-        if 'No results' in response.content:
+        if b'No results' in response.content:
```

Comment:
What's the "b" for?

Reply:
It turns the text in front of it into a bytes literal.
After updating the app to python3, it throws an error `if 'No results' in response.content:
TypeError: a bytes-like object is required, not 'str'`